### PR TITLE
GCE integration

### LINKED ISSUE #278: Support Google GCE
The machine-controller now support GCE. We can look into supporting it here as well

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #278 #318 

```release-note
Added support for GCE
```

### CODE REVIEW

File: examples/terraform/gce/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Terraform
+
+## GCE Provider configuration
+
+### Credentials
+
+Per https://www.terraform.io/docs/providers/google/provider_reference.html#configuration-reference
+ether of the following ENV variables should be accessible:
+* `GOOGLE_CREDENTIALS`
```
Comment (xmudrii) [inline examples/terraform/gce/README.md]: Can we add those variables to [the `environment_variables.md` document](https://github.com/kubermatic/kubeone/blob/master/docs/environment_variables.md)?

File: docs/quickstart-gce.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+# How To Install Kubernetes On GCE Cluster Using KubeOne
+
+In this quick start we're going to show how to get started with KubeOne on GCE.
+We'll cover how to create the needed infrastructure using our example terraform
+configuration and then install Kubernetes. Finally, we're going to show how to
+destroy the cluster along with the infrastructure.
+
+As a result, you'll get Kubernetes 1.13.5 High-Available (HA) clusters with
```
Comment (xmudrii) [inline docs/quickstart-gce.md]: Maybe we can use 1.14.0 as v0.5.0 is going to support it. :)

Comment (kron4eg) [reply docs/quickstart-gce.md]: 👍

File: examples/terraform/gce/README.md (hunk 1 above)
Comment (kron4eg) [reply examples/terraform/gce/README.md]: 👍

**Pull request merged**